### PR TITLE
add Lily Wallet connection instructions

### PR DESCRIPTION
**Pull request by KayBeSee**
Adds connection instructions for Lily Wallet (RPC). Based off a2f764aee3eaeaa15033451521787b912665ae55.

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ import ElectrumAndroid from "../components/ConnectWallet/Wallets/ElectrumAndroid
 import ElectrumDesktop from "../components/ConnectWallet/Wallets/ElectrumDesktop.vue";
 import ElectrumServer from "../components/ConnectWallet/Wallets/ElectrumServer.vue";
 import FullyNoded from "../components/ConnectWallet/Wallets/FullyNoded.vue";
+import LilyWallet from "../components/ConnectWallet/Wallets/LilyWallet";
```
Comment (louneskmt) [inline src/router/index.js:32]:
```suggestion
import LilyWallet from "../components/ConnectWallet/Wallets/LilyWallet.vue";
```
To keep consistency

**Review by louneskmt — COMMENTED** [review]
Thanks for this @KayBeSee! I will test and review this soon.

**Comment by AaronDewes** [comment]
> /umbrel build

Sorry, the bot is offline right now.

**Comment by AaronDewes** [comment]
I'm working on migrating to v2, and @lukechilds will set everything up when 0.3.9 is out.

File: src/components/ConnectWallet/Wallets/LilyWallet.vue [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        Click the and click
+        <span class="font-weight-bold">"Change Data Source"</span> option and
+        then <span class="font-weight-bold">"Connect to Custom Node"</span> to
+        bring up the connection modal.
+      </step>
+      <step>
+        In the <span class="font-weight-bold">"Host"</span>, enter
+        <input-copy
+          class="my-1"
+          :value="`http://${urls.bitcoin.rpc.address}`"
```
Comment (louneskmt) [inline src/components/ConnectWallet/Wallets/LilyWallet.vue:23]:
```suggestion
          :value="`http://${urls.bitcoin.rpc.address}:${urls.bitcoin.rpc.port}`"
```

**Review by louneskmt — CHANGES_REQUESTED** [review]
Awesome @KayBeSee, almost good to be merged! Just a little thing: you forgot to display the RPC port as well. The port can be seen on the screenshots on your documentation, so you must have just forgotten to push the change.

File: src/components/ConnectWallet/Wallets/LilyWallet.vue (hunk 2 above)
Comment (KayBeSee) [reply src/components/ConnectWallet/Wallets/LilyWallet.vue:23]: I was thinking the same thing when I was making the PR, but it looks like `urls.bitcoin.rpc.address` returns the appended port as well. Screenshot:

Screen Shot 2021-04-19 at 9 36 18 AM

Comment (louneskmt) [reply src/components/ConnectWallet/Wallets/LilyWallet.vue:23]: It does not. See:

https://github.com/getumbrel/umbrel-dashboard/blob/fc3975ecc9f16c8b315be6da9419703ef7d6d259/src/store/modules/bitcoin.js#L21-L27

https://github.com/getumbrel/umbrel-dashboard/blob/fc3975ecc9f16c8b315be6da9419703ef7d6d259/src/store/modules/bitcoin.js#L150-156

And:
https://github.com/getumbrel/umbrel-manager/blob/1635a3a1170c8545ee705dbd2437b1442a83f44a/logic/system.js#L59-L81

Comment (louneskmt) [reply src/components/ConnectWallet/Wallets/LilyWallet.vue:23]: Sure, sorry about that. You can manually update SSHing into your Pi and running:

```
sudo ~/umbrel/scripts/update/update --repo getumbrel/umbrel#v0.3.8
```

**Comment by KayBeSee** [comment]
Interesting...

I ran that script and noticed the following error message:

Screen Shot 2021-04-19 at 10 02 05 AM

This is caused because I had a development version of Lily Wallet installed on my box that isn't a published docker container. So looks like it failed, stopped the rest of the script, but still thought it had updated itself.

I went and uninstalled the app and then re-ran the script and now it looks like everything is working.

I just merged in your suggested commit!

**Comment by lukechilds** [comment]
Thanks so much for this @KayBeSee, tested and working great!

I wanted to make a minor amendment but couldn't push to this branch, we're just about to ship a new release so I pushed it here https://github.com/getumbrel/umbrel-dashboard/pull/338 and merged that so we can get this out in this release.

**Pull request closed without merging**